capsules/core/src/virtualizers/virtual_digest.rs

```diff
@@ -95,6 +95,10 @@ impl<'a, A: digest::Digest<'a, L>, const L: usize> VirtualMuxDigest<'a, A, L> {
 impl<'a, A: digest::Digest<'a, L>, const L: usize> digest::DigestData<'a, L>
     for VirtualMuxDigest<'a, A, L>
 {
+    fn set_data_client(&'a self, _client: &'a dyn ClientData<L>) {
+        unimplemented!()
```

Comment:
Why is it ok for this to be `unimplemented`? Why would, for example, this being a `noop` not be better? (ditto for the other `unimplemented`)

Reply:
Well, we merged the `unimplemented!()` on line 211/223 of the same file, so, for consistency?

The challenge here is we are using the same trait twice, so our trick of putting the set_client in the HIL doesn't work (or at least I don't think it does, since I don't think we can implement `Digest::set_client()` twice, can we?).